test/run-k8s-integration-local.sh

```diff
@@ -17,7 +19,8 @@ make -C ${PKGDIR} test-k8s-integration
 # --staging-image=${GCE_PD_CSI_STAGING_IMAGE} --service-account-file=${GCE_PD_SA_DIR}/cloud-sa.json \
 # --deploy-overlay-name=dev --storageclass-file=sc-standard.yaml \
 # --test-focus="External.Storage" --gce-zone="us-central1-b" \
-# --deployment-strategy=${deployment_strategy}
+# --deployment-strategy=${deployment_strategy} --gke-cluster-version=${gke_cluster_version} \
```

Comment:
shouldn't gke-cluster-version and kube-version be mutually exclusive?

Reply:
in fact lets add some validation in the flag parsing to make these mutually exclusive.